### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by ssaviak**
[DEMO LINK](https://ssaviak.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

**Review by vSainiuk — APPROVED** [review]
Great!

File: src/App.jsx [hunk 1]
```diff
@@ -1,107 +1,19 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import { PostList } from './components/PostList';
+import { getCommentsByPostId, getUserById } from './utils';
```
Comment (olya-shyian) [inline src/App.jsx:5]: you did `utils` folder) impressive!👍

File: src/components/CommentList/CommentList.jsx [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo key={comment.id} comment={comment} />
+    ))
+    }
+  </div>
```
Comment (olya-shyian) [inline src/components/CommentList/CommentList.jsx:11]: fix formatting to 
```
<div className="CommentList">
  {comments.map(comment => (
    <CommentInfo key={comment.id} comment={comment} />
  ))}
</div>
```

File: src/components/PostInfo/PostInfo.jsx [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {
+        post.comments.length !== 0
```
Comment (olya-shyian) [inline src/components/PostInfo/PostInfo.jsx:26]: just use `comments` from your destructuring without `post.`

File: src/components/PostList/PostList.jsx [hunk 4]
```diff
@@ -1,3 +1,10 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo key={post.id} post={post} />
+    ))
+    }
```
Comment (olya-shyian) [inline src/components/PostList/PostList.jsx:8]: here the same with formatting

**Review by olya-shyian — CHANGES_REQUESTED** [review]
Well done!🔥

**Review by YegorKorenkov — APPROVED** [review]
Well done! Good job!

**Pull request closed without merging**